Gemfile.lock

```diff
@@ -133,6 +135,15 @@ GEM
       representable (~> 3.0)
       retriable (>= 2.0, < 4.0)
       signet (~> 0.9)
+    google-cloud-core (1.3.0)
```

Comment:
@joshdholtz this is coming from `fastlane`, huh?

Reply:
Ahh yeah, `google-cloud-core` is a [dependency](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-storage/Gemfile#L5) of `google-cloud-storage` which is used in fastlane [here](https://github.com/fastlane/fastlane/blob/master/fastlane.gemspec#L98).

Reply:
Ohh good to know, thanks for confirming!